anmykh/checkio/checkio_4.py

```diff
@@ -0,0 +1,17 @@
+# Let's make our function func more usual. Let it take some argument arg.
+# Return the argument value without any changes.
+import pytest
+
+
+def func(arg):
+    return arg
+
+
+@pytest.mark.parametrize("arg", [
+    "awdac",
+    2,
+    3231,
+    ["as", "sa", "sad"]
+])
+def test_func(arg):
+    assert func(arg) == arg
```

Comment:
better to move tests sepparately,

also you can add assertion in this case:

assert func(arg) is arg

since - what you send is what you get